core/src/main/scala-2/caliban/schema/ArgBuilderDerivation.scala

```diff
@@ -5,8 +5,7 @@ import caliban.InputValue
 import caliban.Value._
 import caliban.schema.Annotations.GQLDefault
 import caliban.schema.Annotations.GQLName
-import magnolia._
-import mercator.Monadic
+import magnolia1._
```

Comment:
Really like the softwaremill versions their packages like this, it's so helpful when you have lagging dependencies

Reply:
💯 agreed. We cross publish some internal deps for both Scala 2&3 and this has been a godsend.